wasm-schema-sandbox/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "wasm-ion-schema"
+version = "0.1.0"
+authors = ["Amazon Ion Team <ion-team@amazon.com>"]
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+default = ["console_error_panic_hook"]
+
+[dependencies]
+wasm-bindgen = "0.2.82"
+ion-schema = { path="../../ion-schema-rust" }
+
+# The `console_error_panic_hook` crate provides better debugging of panics by
+# logging them with `console.error`. This is great for development, but requires
+# all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
+# code size when deploying.
+console_error_panic_hook = { version = "0.1.6", optional = true }
+
+# `wee_alloc` is a tiny allocator for wasm that is only ~1K in code size
+# compared to the default allocator's ~10K. It is slower than the default
+# allocator, however.
+#
+# Unfortunately, `wee_alloc` requires nightly Rust when targeting wasm for now.
+wee_alloc = { version = "0.4.5", optional = true }
```

Comment:
minor comment—it would be worth considering whether we can get away with the regular allocator so that we don't need "nightly" Rust. It's not essential now, but before we release v1.0.0, it would be a good idea to try to get onto "stable" Rust.